src/pip/_internal/commands/completion.py

```diff
@@ -80,7 +83,9 @@ def run(self, options, args):
         shell_options = ['--' + shell for shell in sorted(shells)]
         if options.shell in shells:
             script = textwrap.dedent(
-                COMPLETION_SCRIPTS.get(options.shell, '')
+                COMPLETION_SCRIPTS.get(options.shell, '').format(
```

Comment:
Why not just simply do a `%` formatting; for consistency?

Reply:
Great suggestion, thanks.

Reply:
:)